c2rust-analyze/src/main.rs

```diff
@@ -454,6 +454,7 @@ fn run(tcx: TyCtxt) {
             assert_eq!(local, l);
         }
 
+        let mut const_ref_tys = HashMap::new();
```

Comment:
why not insert into the `acx` map directly?

Reply:
There'd be multiple mutable references to `acx` then, due to `acx.rvalue_tys`..  It'd probably be better to construct the `AnalysisCtxt` with the maps already constructed, but that'd be a bit more of a change, so I just did it this way for now.

Reply:
Oh, well it might not have been that specifically then, but there definitely was a borrow checker error.  I tried this way first but it didn't work.